src/shared/check_outdated_data.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+int Run(int maxTimeDiff, char *aavild_data) {
```

Comment:
This function needs to change. `Run` is **not** a good function name here. Maybe `check_outdated_data` instead?
aavild_data is also not a good name. What is the data and what does it have to do with aavild? aavild is a person not data!

Reply:
I will change it sir, please dont hurt me. I will give 2 rupees